modules/enable-openshift-monitoring-central-helm.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-openshift-monitoring-central-helm_{context}"]
+= Monitoring Central services by using Helm charts
+
+[role="_abstract"]
+You can monitor  Central and Central services, by changing the configuration options in the `central-services` Helm chart.
+
+.Procedure
+. Update the `values-public.yaml` configuration file with the following values:
++
+[source,yaml]
+----
+monitoring.openshift.enabled: true
```

Comment:
This is already the default. Do we need this section?

Reply:
I think we should keep it. It's easier to reason how to disable it if you see how it's enabled.

Reply:
Maybe we can condense it at least, as mentioned above I think we over-document now.